components/script/stylesheet_loader.rs

```diff
@@ -92,6 +97,24 @@ impl FetchResponseListener for StylesheetContext {
 
     fn process_response(&mut self,
                         metadata: Result<FetchMetadata, NetworkError>) {
+        match metadata {
+            Ok(FetchMetadata::Unfiltered(_)) => {
+                self.origin_clean.set(true);
```

Comment:
Since it starts as true, we only need to set it in the cases where it's not origin clean.